process/block/baseProcess.go

```diff
@@ -794,6 +794,15 @@ func (bp *baseProcessor) prepareDataForBootStorer(args bootStorerDataArgs) {
 		EpochStartTriggerConfigKey: args.epochStartTriggerConfigKey,
 	}
 
+	log.Debug("baseProcessor.prepareDataForBootStorer",
```

Comment:
this is called in every commit block - is it not too much data for log.Debug ?

Reply:
yeah, was used in debugging

Reply:
removed